modules/nw-configuring-qinq-sriov-proc.adoc

```diff
@@ -0,0 +1,243 @@
+// Module included in the following assemblies:
+//
+// * 
+// * 
+// * 
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="nw-configuring-qinq-sriov-proc_{context}"]
+= Configuring QinQ support for SR-IOV enabled workloads
+
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc``).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the SR-IOV Network Operator.
+
+.Procedure
+
+. To verify that the SR-IOV Operator is ready, run the following command and view the resulting output:
++
+[source,terminal]
+----
+$ oc get csv -n openshift-sriov-network-operator
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                                          DISPLAY                   VERSION               REPLACES   PHASE
+sriov-network-operator.v4.15.0-202401261531   SR-IOV Network Operator   4.15.0-202401261531              Succeeded
+----
+
+. To verify that the SR-IOV pods are deployed, run the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-sriov-network-operator
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                                      READY   STATUS    RESTARTS   AGE
+network-resources-injector-drcmj          1/1     Running   0          9m37s
+operator-webhook-5s5vd                    1/1     Running   0          9m37s
+sriov-network-config-daemon-zv2lj         1/1     Running   0          9m37s
+sriov-network-operator-6c5dbb89c8-lmd5s   1/1     Running   0          9m53s
+----
+
+. Check the machine config pool:
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME         CONFIG                                                 UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
+master       rendered-master-acd1358917e9f98cbdb599aea622d78b       True      False      False      3              3                   3                     0                      22h
+worker-cnf   rendered-worker-cnf-1d871ac76e1951d32b2fe92369879826   False     True       False      2              1                   1                     0                      22h
```

Comment:
I would omit all the above. Instead, you can simply say, refer to the installation section. No need to repeat all that. I just put in the prerequisites, sriov network operator installed.